src/unix/linux_like/mod.rs

```diff
@@ -264,7 +271,7 @@ cfg_if! {
         }
         impl Eq for epoll_event {}
         impl ::fmt::Debug for epoll_event {
-            fn fmt(&self, f: &mut ::fmt::Formatter) -> ::fmt::Result {
+            fn fmt(&self, f: &mut ::fmt::Formatter<'_>) -> ::fmt::Result {
```

Comment:
And I think this also won't as I mentioned on the issue.

Reply:
I added explicit lifetimes to `Formatter` args.